README.md

```diff
@@ -64,6 +64,21 @@ http://localhost/user/reset/1/123456789/ai6u4-iY1LgZFUjwVW2uXjh5jblqgsfUHGFS_U/l
 
 **Note: This is a ONE-TIME login. You'll automatically be logged in during future uses. However, if you ever reset your container, you will have to redo this process.**
 
+## Lint Guidelines (SCSS)
+If you make any changes to the `scss` files, make sure to lint them before finalizing your pull request.
+
+You can run the following commands to `check` for errors:
```

Comment:
Let's add the context that this should be run in the "node" container. Maybe just add "In the `node` container" to the beginning of this sentence. It's obvious when you think about it, but ...